src/main/java/com/microsoft/azure/spring/data/cosmosdb/core/convert/DateConversionConstants.java

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.data.cosmosdb.core.convert;
+
+public class DateConversionConstants {
+
+    public static final String ISO_8601_COMPATIBLE_DATE_PATTERN = "yyyy-MM-dd'T'HH:mm:s:SSSXXX";
```

Comment:
we have one class named `Constants` already, looks you can put the String there.